### DISCUSSION TITLE
Best configuration for slow hardware (raspberry pi 3, old iPhone client)

### DISCUSSION THREAD

**Discussion by sergueif** [navidrome/navidrome#2248] [Q&A]
Hello, what are all the options/dials/switches to set so navidrome is as light as possible on the slow hardware? 

A slight majority of the time, it runs on my rpi3 just fine, but sometimes I catch it chugging and puffing doing something with `ffmpeg` and high cpu on `navidrome` itself. It then overheats and restarts my rpi3. The old iPhone client is also hella slow (using Substreamer app I think).

Anyway, can I tell navidrome all around to just do the bare minimum and not do much transcoding and stuff?

**Comment by kgarner7** [comment]
The main option I'd recommend changing is `ScanSchedule`, which is normally set to every one minute (you can make it less frequent, and if you don't add music often, can even disable it). If you're not using them, you can also disable Listenbrainz and Last.fm.
As for transcoding, I _think_ you should be able to accomplish disabling this by deleting all transcoding configurations.

**Comment by sergueif** [comment]
Couldn't find a way to turn off transcoding, but I set the `ffmpeg` path to `/dev/null` :) we'll see how it goes.

Seems to be running a bit better now with the lower ScanSchedule, thank you
